packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3014,6 +3029,34 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     ) {
         this.verifyNotClosed();
         assert(this.connected, 0x259 /* "Container disconnected when trying to submit system message" */);
+
+        if (new TextEncoder().encode(JSON.stringify(contents)).length > 2000) {
```

Comment:
>2000 [](http://example.com/codeflow?start=72&length=4)

I wonder if this should be a container runtime parameter ?

Reply:
I figure adding this as a parameter alongside specifying the compression algorithm makes sense so that it can be configured. Seems like a good idea